src/components/Movies/MovieForm/MovieForm.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from "react";
+import Dialog from "../Dialog/Dialog";
+import "./movieForm.css";
+
+class MovieForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      formData: props.initialMovieInfo || {
+        name: "",
+        year: "",
+        movieUrl: "",
+        vote_average: "",
+        genres: "",
+        runtime: "",
+        overview: "",
+      },
```

Comment:
can use here the default props:
https://blog.logrocket.com/complete-guide-react-default-props/

Reply:
Removed the defaultProps declaration from this.state and added MovieForm.defaultProps.